dev-util/radare2/radare2-1.1.0.recipe

```diff
@@ -93,11 +98,6 @@ INSTALL()
 {
 	make install
 	cp libr/*/libr_*.so $libDir
-
-	mkdir -p `dirname $docDir`
-	mv $prefix/share/doc $docDir
-	rm -rf $prefix/share
```

Comment:
But moving the directory from $prefix/data/doc to $docDir puts the documentation under documentation/packages/radare2/radare2/  

Is this correct documentation directory to place the html/markdown files?